rmw/include/rmw/rmw.h

```diff
@@ -1781,12 +1781,68 @@ RMW_WARN_UNUSED
 rmw_ret_t
 rmw_destroy_client(rmw_node_t * node, rmw_client_t * client);
 
-/// Send a service request to the rmw server
+/// Send a ROS service request.
 /**
- * \param[in] client The connected client over which to send this request
- * \param[in] ros_request the request message to send to the server
- * \param[out] sequence_id A unique identification value to identify this request
- * \return RMW_RET_OK if successful, otherwise an appropriate error code
+ * Send a ROS service request to one or more service servers, with matching QoS policies,
+ * using the given client.
+ *
+ * \note It is implementation defined how many service servers may get, and potentially react to,
+ *   the same request, considering there may be more than one server for the same service
+ *   in the ROS graph.
+ *
+ * On success, this function will return a sequence number.
+ * It is up to callers to keep that sequence number to pair the ROS service request just sent
+ * with future ROS service responses, to be taken using rmw_take_response().
```

Comment:
```suggestion
 * It is up to callers to save the returned sequence number to pair the ROS service request just sent
 * with future ROS service responses (taken using rmw_take_response()).
```

(I'm not sure this is much better, but I found it a bit easier to read)